requirements.txt

```diff
@@ -0,0 +1,3 @@
+overpy==0.3.1
+transitfeed==1.2.14
+osmapi==0.6.2
```

Comment:
Is there any reason to require exact versions here?
I'd say we replace the `==` with `>=` because it is very likely that things will continue to work with future versions of these libraries.